pkg/controller/provider/container/vsphere/collector.go

```diff
@@ -715,7 +715,7 @@ func (r *Collector) propertySpec() []types.PropertySpec {
 				fExtraConfig,
 				fGuestName,
 				fGuestID,
-				fTpmPresent,
+				//fTpmPresent,
```

Comment:
This value isnt used later on  in the flow of the creation? I mean wont it cause issues in the VM migration if we wont set it with some kind of default value it in the inventory?

Reply:
it won't be reported by vsphere so from our side it will appear as if the vm doesn't have tpm in the rest of the flow